### DISCUSSION TITLE
Claim free JCB package key, not working with "watch"

### DISCUSSION THREAD

**Discussion by phlambotte** [joomengine/Joomla-Component-Builder#960] [General]
Hello, I tried to obtain a package key following the instructions in https://www.joomlacomponentbuilder.com/how-to-get-a-free-jcb-package-key
I was not working if I "watch" a repository, but it works fine when I select "star" for a repository.
Hope this help somebody.

**Comment by Llewellynvdm** [comment]
This area of JCB needs help... as mentioned here: https://git.vdm.dev/joomla/Component-Builder/issues/1049#issuecomment-26882

We are working on JCB to fix all known issues around the JCB packages, and will keep you posted with out progress.
